winlogbeat/docs/getting-started.asciidoc

```diff
@@ -70,11 +72,15 @@ logging:
   level: info
 --------------------------------------------------------------------------------
 
+For instructions on how to obtain a list of available event logs see the
+configuration details for
+<<configuration-winlogbeat-options-event_logs-name,event_logs.name>>.
```

Comment:
Looks right to me. You really only need to specify the first part, though, unless you want the link text to differ from the heading text. So you could have just:
<<configuration-winlogbeat-options-event_logs-name>> 

But I'd just leave it as it is.

BTW the doc looks good. If you ever decide you want to be a technical writer, hit us up. :-)